contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-rule.y

```diff
@@ -2747,7 +2747,7 @@ tsql_CreateTrigStmt:
 					 * trigger will be created as part of
 					 * this create trigger command.
 					 */
-					n1->funcname = list_make1(makeString(n1->trigname));
```

Comment:
The overall logic looks fine but I am wondering why we are treating `funcname` and `trigname` differently? Due to this only we introduced this issue in the first place. IMO let's directly assign (or duplicate the list) `n1->funcname = n2->trigname` somewhere at the end since both of them should be same at the end.

Reply:
We can't because trigname is char* only, but funcname takes list of char*.